### ISSUE TITLE
Problem with erl_ssl_path fact on RHEL 7.x, erlang 18.3

### ISSUE THREAD

**Issue by bostrowski13** [voxpupuli/puppet-rabbitmq#610]
When running the puppet agent, I am receiving the following error regarding the "erl_ssl_path" custom fact.

```
[root@dwrabbitmq02 bostrowski]# puppet agent -t
Info: Using configured environment 'production'
Info: Retrieving pluginfacts
Info: Retrieving plugin
Info: Loading facts
Error: Facter: error while resolving custom fact "erl_ssl_path": undefined method `with_env' for Facter::Util::Resolution:Class
Info: Caching catalog for <REMOVED>
```

While this doesn't prevent the catalog from being run, it is an error.

**Comment by wyardley** [comment]
@bostrowski13 can you see if #609 (by @costela, just merged) fixes it?

**Comment by wyardley** [comment]
Closed via #609
